plugin/controller/lib/impl/mcp/MCPControllerRegister.ts

```
@@ -0,0 +1,450 @@
+    this.router = app.router;
+
+    this.controllerMeta = controllerMeta;
+  }
+
+  static addHook(hook: MCPControllerHook) {
+    MCPControllerRegister.hooks.push(hook);
+  }
+
+  async mcpStreamServerInit() {
+    const allRouterFunc = this.router.all;
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)